compiler/rustc_mir_transform/src/coverage/mod.rs

```diff
@@ -498,6 +493,14 @@ fn make_code_region(
     span: Span,
     body_span: Span,
 ) -> CodeRegion {
+    debug!(
+        "Called make_code_region(file_name={}, source_file={:?}, span={}, body_span={})",
+        file_name,
+        source_file,
+        source_map.span_to_diagnostic_string(span),
+        source_map.span_to_diagnostic_string(body_span)
+    );
+
```

Comment:
You can use `#[instrument(level="debug", skip(source_map))]` attribute macro.

Reply:
Note that this isn't new code; I'm just moving it here as-is so that it's out of the way of other changes to `inject_coverage_span_counters` (which is the only caller).